packages/datasource/src/api/orgs/enroll/index.js

```diff
@@ -0,0 +1,48 @@
+import client from '../../client';
+import { sendCode } from './sendCode';
+import { uploadIncumbencyCert } from './uploadIncumbencyCert';
+
+const enroll = ({
+  name,
+  company,
+  industry_type,
+  member_range,
+  job_title,
+  company_base,
+  contact_phone,
+  audit_type,
+  email,
+  email_code,
+  incumbency_cert,
```

Comment:
For a JS application, it's more common to use camel cases, and I also understand our Python-based API prefers snake cases. So I was thinking to encapsulate a helper method to convert all the params from camelCase to snake_case or do it inversely. It's not a priority, just thinking we may refactor it later.